### PR TITLE
Add warning for Quick Edit Mode

### PR DESCRIPTION
**Pull request by Grauldon**
[SpongeDocs](https://github.com/SpongePowered/SpongeDocs/issues/797)

This PR adds a warning to the [Creating A Launch Script](https://docs.spongepowered.org/stable/en/server/getting-started/launch-script.html) page of Getting Started in the SpongeDocs. It also provides steps to disabling the `Quick Edit Mode` in Command Prompt and PowerShell shells.

Signed-off-by: Grauldon <17791090+Grauldon@users.noreply.github.com>

### CODE REVIEW

**Comment by Grauldon** [comment]
I forgot to mention @ItsDoot and @Brycey92 for review.

File: source/server/getting-started/launch-script.rst [hunk 1]
```diff
@@ -30,6 +30,26 @@ Windows
 
 Save your Windows launch script as ``launch.bat``.
 
+.. warning::
+   Launching your server in a Command Prompt or PowerShell environment can lead to the server randomly hanging. During
```
Comment (phit) [inline source/server/getting-started/launch-script.rst]: it doesn't randomly hang, it happens when a user marks something

**Comment by phit** [comment]
I still don't think this is needed, there's hundreds of different shells all which may add their own way to freeze a process. Or have other oddities that may impact how the shell works.

**Comment by Spongy** [comment]
A preview for this pull request is available at https://cdn.rawgit.com/Spongy/SpongeDocs-PRs/9bb37eb/.

Here are some links to the pages that were modified:

- modified: https://cdn.rawgit.com/Spongy/SpongeDocs-PRs/9bb37eb/server/getting-started/launch-script.html

_Since the preview frequently changes, please link to [this comment](https://github.com/SpongePowered/SpongeDocs/pull/809#issuecomment-502472379), not to the direct url to the preview._

**Comment by ST-DDT** [comment]
There might be hundreds of shells out there, but cmd/powershell are the most commonly used ones on windows. 
I'm not sure whether it should be a warning or just an info. Or maybe rephrase it to a recommendation? (Not sure about that)
Does Minecraft itself or Forge have a similar entry on their pages?

**Comment by phit** [comment]
in my opinion the warning box is enough, we don't need to explain how to disable it. 
> Does Minecraft itself or Forge have a similar entry on their pages?

neither do disable the default server GUI

**Comment by Grauldon** [comment]
So, delete the `Disabling Quick Edit Mode` section completely and change the Warning from:

Launching your server in a Command Prompt or PowerShell environment can lead to the server ~randomly~ hanging. During this time, the only messages captured in the log state the server skipped many seconds or minutes worth of ticks. The cause is the `Quick Edit Mode` of these shells. This mode freezes the process when you mark something for a quick edit. ~You can disable the `Quick Edit Mode` in the Properties dialogue to prevent the hangs from occurring.~

to

Launching your server in a Command Prompt or PowerShell environment can lead to the server hanging. During this time, the only messages captured in the log state the server skipped many seconds or minutes worth of ticks. The cause is the Quick Edit Mode of these shells. This mode freezes the process when you mark (or highlight) something for a quick edit (e.g., copy/paste). You can prevent hangs from occurring by **not** highlighting any text on the screen or by disabling the `Quick Edit Mode` in the Properties dialogue.

File: source/server/getting-started/launch-script.rst (hunk 1 above)
Comment (Grauldon) [reply source/server/getting-started/launch-script.rst]: Removed randomly. Waiting on more feedback before pushing a commit. Thanks for the clarification!

File: source/server/getting-started/launch-script.rst [hunk 2]
```diff
@@ -30,6 +30,13 @@ Windows
 
 Save your Windows launch script as ``launch.bat``.
 
+.. warning::
+   Launching your server in a Command Prompt or PowerShell environment can lead to the server hanging. During this 
+   time, the only messages captured in the log state the server skipped many seconds or minutes worth of ticks.
```
Comment (Inscrutable) [inline source/server/getting-started/launch-script.rst]: This sentence is a little hard to parse. May I suggest changing it to read
...`messages captured in the log will state that the server skipped`...

File: source/server/getting-started/launch-script.rst [hunk 3]
```diff
@@ -30,6 +30,13 @@ Windows
 
 Save your Windows launch script as ``launch.bat``.
 
+.. warning::
+   Launching your server in a Command Prompt or PowerShell environment can lead to the server hanging. During this 
+   time, the only messages captured in the log state the server skipped many seconds or minutes worth of ticks. 
+   The cause is the ``Quick Edit Mode`` of these shells. This mode freezes the process when you mark (or highlight) 
+   something for a quick edit (e.g., copy/paste). You can prevent hangs from occurring by not highlighting any text on 
+   the screen or by disabling the ``Quick Edit Mode`` in the Properties dialogue.
```
Comment (Inscrutable) [inline source/server/getting-started/launch-script.rst]: Comma needed after `screen`.

File: source/server/getting-started/launch-script.rst [hunk 4]
```diff
@@ -30,6 +30,13 @@ Windows
 
 Save your Windows launch script as ``launch.bat``.
 
+.. warning::
+   Launching your server in a Command Prompt or PowerShell environment can lead to the server hanging. During this 
+   time, the only messages captured in the log state the server skipped many seconds or minutes worth of ticks. 
+   The cause is the ``Quick Edit Mode`` of these shells. This mode freezes the process when you mark (or highlight)
```
Comment (Inscrutable) [inline source/server/getting-started/launch-script.rst]: This might as well be one sentence, if you change `This mode` to `which`.

File: source/server/getting-started/launch-script.rst [hunk 5]
```diff
@@ -30,6 +30,13 @@ Windows
 
 Save your Windows launch script as ``launch.bat``.
 
+.. warning::
+   Launching your server in a Command Prompt or PowerShell environment can lead to the server hanging. During this 
+   time, the only messages captured in the log state the server skipped many seconds or minutes worth of ticks. 
+   The cause is the ``Quick Edit Mode`` of these shells. This mode freezes the process when you mark (or highlight) 
+   something for a quick edit (e.g., copy/paste). You can prevent hangs from occurring by not highlighting any text on
```
Comment (Inscrutable) [inline source/server/getting-started/launch-script.rst]: I would change `hangs` to `this`, mainly because there are lots of other ways a server might hang ;)

**Review by Inscrutable — COMMENTED** [review]
I came, I saw, I quibbled.

File: source/server/getting-started/launch-script.rst (hunk 2 above)
Comment (Grauldon) [reply source/server/getting-started/launch-script.rst]: Changed

File: source/server/getting-started/launch-script.rst (hunk 3 above)
Comment (Grauldon) [reply source/server/getting-started/launch-script.rst]: Added

File: source/server/getting-started/launch-script.rst (hunk 4 above)
Comment (Grauldon) [reply source/server/getting-started/launch-script.rst]: Changed

File: source/server/getting-started/launch-script.rst (hunk 5 above)
Comment (Grauldon) [reply source/server/getting-started/launch-script.rst]: Changed

**Comment by Brycey92** [comment]
Thanks for the PR! I suggest the following rephrasing for better flow:

Launching your server in a Command Prompt or PowerShell environment can lead to the server hanging due to the Quick Edit Mode of these shells. This mode freezes the process when you highlight something or click inside the console window. During this time, the only messages captured in the log state the server skipped many seconds or minutes worth of ticks. You can prevent hangs from occurring by not highlighting any text on the screen and not clicking inside the window, or by disabling the Quick Edit Mode in the Properties dialogue.

**Comment by Grauldon** [comment]
You're welcome! Thanks for the suggestion. It is a much better flow.

**Review by ST-DDT — APPROVED** [review]

File: source/server/getting-started/launch-script.rst (hunk 2 above)
Comment (Inscrutable) [reply source/server/getting-started/launch-script.rst]: It's reverted in the latest edit :/

File: source/server/getting-started/launch-script.rst (hunk 5 above)
Comment (Inscrutable) [reply source/server/getting-started/launch-script.rst]: This seems to have been reverted too.

**Comment by Grauldon** [comment]
Oops! I didn't think to check this. I need to slow down and smell the roses! :)
